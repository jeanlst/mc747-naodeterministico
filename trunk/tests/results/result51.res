Semantic Error
A string nao tem tamanho compativel com o vetor
