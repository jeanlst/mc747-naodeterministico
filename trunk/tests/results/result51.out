Semantic Error
Tipos incompativeis para a atribuicao
